docs/BR.md

```diff
@@ -1190,15 +1190,36 @@ The certificate request MAY include all factual information about the Applicant
 
 Applicant information MUST include, but not be limited to, at least one Fully-Qualified Domain Name or IP address to be included in the Certificate's `subjectAltName` extension.
 
-[Section 6.3.2](#632-certificate-operational-periods-and-key-pair-usage-periods) limits the validity period of Subscriber Certificates. The CA MAY use the documents and data provided in [Section 3.2](#32-initial-identity-validation) to verify certificate information, or may reuse previous validations themselves, provided that the CA obtained the data or document from a source specified under [Section 3.2](#32-initial-identity-validation) or completed the validation itself no more than 825 days prior to issuing the Certificate. For validation of Domain Names and IP Addresses according to Section 3.2.2.4 and 3.2.2.5, any data, document, or completed validation used MUST be obtained no more than 398 days prior to issuing the Certificate.
+[Section 6.3.2](#632-certificate-operational-periods-and-key-pair-usage-periods) limits the validity period of Subscriber Certificates.
+
+The CA MAY use the documents and data provided in [Section 3.2](#32-initial-identity-validation) to verify certificate information, or may reuse previous validations themselves, provided that the CA obtained the data or document from a source specified under [Section 3.2](#32-initial-identity-validation) or completed the validation itself within the maximum number of days prior to issuing the Certificate, as defined in the following table:
+
+Table: Maximum Allowed Non-SAN Validation Data Reuse Period
+
+| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for data reuse__ |
+| --                                 | -                             | -                                         |
+|                                    | September 15, 2025            | 825                                       |
+| September 15, 2025                 | September 15, 2026            | 366                                       |
```

Comment:
suggestion: either remove the "certificate issued before" date from the last row, or add more rows which further reduce data reuse periods on dates in lock-step with the other tables below. Otherwise it's unclear what happens after Sept 15 2026.
```suggestion
|                                    | September 15, 2025            | 825                                       |
| September 15, 2025                 |                               | 366                                       |
```
```suggestion
|                                    | September 15, 2025            | 825                                       |
| September 15, 2025                 | September 15, 2026            | 366                                       |
| September 15, 2026                 | September 15, 2027            | 200                                       |
| September 15, 2027                 |                               | 100                                       |
```

Reply:
I interpreted it to mean "no reuse of non-SAN validation after September 15, 2026," which I am hoping is not the case.

Reply:
> I interpreted it to mean "no reuse of non-SAN validation after September 15, 2026," which I am hoping is not the case.

@clintwilson could you clarify this point please? What is supposed to happen after 15.09.2026?